src/assessment.handlebars

```diff
@@ -606,25 +608,23 @@
+          <p>The assessor looks at how your disability or health conditions affect the way you carry out a number of different activities. These activities fall under the categories <strong>Daily living</strong> and <strong>Mobility</strong>.</p>
           <p>You will be given a score for each of these activities.</p>
           <p>For example, for the activity &#x201c;dressing and undressing&#x201d;, if you can dress and undress unaided, you will score 0 points. If you cannot dress or undress at all, this will score 8 points. If you need prompting to select appropriate
-            clothing, or if you need assistance to undress part of your body, you will get a score in between.</p>
+            clothing, or if you need assistance to undress part of your body, you will get a score in between. Some other activities, such as moving around, have the highest scores of 10 or 12.</p>
           <p>Your scores across the activities will be added, and that will decide the recommendation the assessor makes.</p>
           <div class="about-buttons-content">
           </div>
         </div>
-        <h2><button aria-controls="how-to-answer" aria-expanded="true">How to answer the questions</button></h2>
+        <h2><button aria-controls="how-to-answer" aria-expanded="true">How to answer the assessment questions</button></h2>
         <div id="how-to-answer" aria-hidden="false">
-          <p>Here are some tips for answering the questions on the form and at the face to face assessment:</p>
           <ul class="ticks">
-            <li>Don&#x2019;t put on a brave face when talking about your disability or health condition; be clear about how much it affects you.</li>
-            <li>Your condition may change from day to day, be clear about how it impacts on you on your worst day.</li>
-            <li>Be aware of what it means to say you are able to carry out a task. Can you do it safely, properly, more than once, and within a reasonable time period? Does it cause you pain when you carry out the task? Make it clear if you use any aids to
-              help you. Does carrying out a task&#x2F; activity exhaust you afterwords?</li>
+            <li>Don&#x2019;t put on a brave face when talking about your disability or health condition - be clear about how much it affects you.</li>
+            <li>Your condition may change from day to day, be clear about how it impacts on you on your worst day and other days when you don’t feel well.</li>
+            <li>Be aware of what it means to say you are able to carry out a task. Can you do it safely, properly, more than once, and within a reasonable time period? Does it cause you pain when you carry out the task? Make it clear if you use any aids to help you. Does carrying out a task/activity exhaust you afterwards?</li>
```

Comment:
I think we should change this:
'Does carrying out a task exhaust you afterwards?' Doesn't make sense.